javascript/javascript1/week3/homework/main-3.js

```diff
@@ -0,0 +1,59 @@
+const seriesDurations = [
+  {
+    title: "Game of thrones",
+    days: 3,
+    hours: 1,
+    minutes: 0,
+  },
+  {
+    title: "Sopranos",
+    days: 3,
+    hours: 14,
+    minutes: 0,
+  },
+  {
+    title: "The Wire",
+    days: 2,
+    hours: 12,
+    minutes: 0,
+  },
+];
+
+seriesDurations.push({
+  title: "Dark",
+  days: 0,
+  hours: 22,
+  minutes: 58,
+});
+seriesDurations.push({
+  title: "Castle",
+  days: 5,
+  hours: 9,
+  minutes: 45,
+});
+seriesDurations.push({
+  title: "Behind Her Eyes",
+  days: 0,
+  hours: 5,
+  minutes: 0,
+});
+seriesDurations.splice(0, 3);
+console.log(seriesDurations);
+
```

Comment:
Cool ! You could have also manually replace the example series with your favorite ones, but this is works great 🏋️‍♀️

Reply:
Thanks:)